locales/en/yourloops.json

```diff
@@ -157,10 +158,11 @@
   "signup-steppers-step3": "Create profile",
   "signup-steppers-step4": "Complete Signup",
   "signup-steppers-step-unknown": "Unknown step",
-  "signup-steppers-ending-message": "Please follow the link in the email we just sent you at {{email}} to verify and activate your account",
+  "signup-steppers-ending-message": "Please follow the link in the email we just sent you at {{email}} to verify and activate your account\nIf you don’t receive any email from us, please contact our support at technical.support@diabeloop.com\n",
```

Comment:
The `\n` (line return) is ignored in HTML.
We shouldn't have it here.
If you want a line return, you should use the `<br />` tag.
You have to either:
- Split this translation in 2 part
- Use a `<Trans />` component.

Reply:
as i have several message that required to include <trans> tag, i propose to do a PR that will contains this scope as we discussed.